format/Message.fbs

```diff
@@ -64,8 +64,11 @@ table Date {
 table Time {
 }
 
+enum TimeUnit: short { MILLISECOND, MICROSECOND }
```

Comment:
Can we add nanosecond? This is quite common in the financial world for example (part of the reason that pandas uses nanoseconds)